app/MasterTemplate.hs

```diff
@@ -38,7 +38,8 @@ masterTemplate title headers body scripts =
 header :: T.Text -> H.Html
 header page =
     H.nav ! A.class_ "row header" $ do
-        H.img ! A.id "courseography-header" ! A.src "/static/res/img/logo.png"
+        H.a ! A.href "https://courseography.cdf.toronto.edu/graph" $ do
```

Comment:
You should use a relative path, just like the other links in the nav bar. (This way the link will work on localhost too, for example.) So in this case it's just `/graph`.